pandas/tests/frame/test_alter_axes.py

```diff
@@ -979,7 +979,7 @@ def test_reset_index(self, float_frame):
         ):
             values = lev.take(level_codes)
             name = names[i]
-            tm.assert_index_equal(values, Index(deleveled[name]))
+            tm.assert_index_equal(values, Index(deleveled[name]), check_names=False)
```

Comment:
why is this changed?

Reply:
``lev.take(level_codes)`` doesn't provide a name any more, while a rest index does provides its Series with a name (as it should.

I've added a test ``assert values.name is None`` to make this more explicit.